x-pack/plugins/ml/public/application/model_management/add_model_flyout.tsx

```diff
@@ -110,6 +110,7 @@ export const AddModelFlyout: FC<AddModelFlyoutProps> = ({ onClose, onSubmit, mod
               key={tab.id}
               isSelected={selectedTabId === tab.id}
               onClick={setSelectedTabId.bind(null, tab.id)}
+              data-test-subj={`mlAddTrainedModelFlyoutTab-${normalize(tab.id)}`}
```

Comment:
Tab IDs are `clickToDownload` and `manualDownload`. Any particular reason to `normalize` here? There no spaces in the strings as well. I reckon it's ok to use as-is